tests/GeneralStateTestsRunner.js

```diff
@@ -126,7 +126,11 @@ function runTestCase (options, testData, t, cb) {
 
 module.exports = function runStateTest (options, testData, t, cb) {
   try {
-    const testCases = parseTestCases(options.forkConfig, testData, options.data, options.gasLimit, options.value)
+    let aliasForkConfig
+    if (options.forkConfig === 'Petersburg') {
```

Comment:
Some note here: when I added the fork config logic here to the tests some time ago I added it in a way that both capital and non-capital names can be used, see e.g. [here](https://github.com/ethereumjs/ethereumjs-vm/blob/master/tests/GeneralStateTestsRunner.js#L58), so `e.g. `Byzantium` or `byzantium`. Still not sure if this was a good idea, I actually already did a first take to remove this lately along some other PR and then stumbled over some unexpected stuff which made this a bit more complicated than expected.

So I would suggest also for now that we keep that behavior and concentrate on the `Petersburg` changes. But to keep things consistent we should then also do these two `Petersburg` comparisons in the PR case-agnostic and compare the lowercase versions.

Reply:
I have made changes in 948fcfa8f4e5e8269e5d8cc55dc02a26b8655af7 such that the fork config for tests can be provided 'petersburg' or 'Petersburg'.

However, I noticed that `node ./tests/tester -s --fork="Byzantium"` works but node ./tests/tester -s --fork="byzantium" does not. This is because the `ethereum/tests` assume that the fork names will be written in upper case. The 'petersburg'/'Petersburg' case is not affected because I've made the name check in the added hardfork alias logic case insensitive.

One way to address this would be the changes I pushed (with another branch) here: https://github.com/ethereumjs/ethereumjs-vm/compare/add-petersburg-aka-constantinople-fix-support...fix-test-fork-name-case-handling